backend/molgenis-emx2-sql/src/main/resources/org/molgenis/emx2/sql/migration6.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "MOLGENIS"."schema_metadata"
+    ADD COLUMN IF NOT EXISTS is_changelog_enabled BOOLEAN DEFAULT FALSE ;
```

Comment:
why didn't we simply keep on using 'settings' for this? I.e. in what cases should we use settings and in what cases should change the metadata model?